docs/sources/reference/components/otelcol/otelcol.exporter.loadbalancing.md

```diff
@@ -62,10 +62,18 @@ Name          | Type     | Description                          | Default     |
 `routing_key` | `string` | Routing strategy for load balancing. | `"traceID"` | no
 
 The `routing_key` attribute determines how to route signals across endpoints. Its value could be one of the following:
-* `"service"`: spans with the same `service.name` will be exported to the same backend.
+- `"service"`: spans/logs/metrics with the same `service.name` will be exported to the same backend.
 This is useful when using processors like the span metrics, so all spans for each service are sent to consistent {{< param "PRODUCT_NAME" >}} instances
 for metric collection. Otherwise, metrics for the same services would be sent to different instances, making aggregations inaccurate.
-* `"traceID"`: spans belonging to the same traceID will be exported to the same backend.
+- `"traceID"`: spans/logs belonging to the same traceID will be exported to the same backend.
+- `"resource"`: metrics belonging to the same resource will be exported to the same backend.
+- `"metric"`: metrics with the same name will be exported to the same backend
+- `"streamID"`: metrics with the same streamID will be exported to the same backend
+
+The loadbalancer will configure the exporter for the singal types supported by the `routing_key`
+
+Metrics support in `otelcol.exporter.loadbalancing` is considered experimental, so an exporter
+will be configured for metrics only if Alloy is run with `--stability-level=experimental`
```

Comment:
```suggestion
> **EXPERIMENTAL**: Metrics support in `otelcol.exporter.loadbalancing` is an [experimental][] feature.
> Experimental features are subject to frequent breaking changes, and may be removed with no equivalent replacement.
> The `stability.level` flag must be set to `experimental` to use the feature.
```

The way this is presented may change, but for now this matches the general notice of an Experimental feature here: https://grafana.com/docs/alloy/latest/reference/stdlib/array/#arraycombine_maps  This is parallel to the notice on Experimental or Public preview components (example:  https://grafana.com/docs/alloy/latest/reference/components/beyla/beyla.ebpf/ )

Reply:
Thanks for the documentation fixes.